### PR TITLE
New package: zssh

### PR DESCRIPTION
**Pull request by ufud-org**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl (cross-build)
  - armv6l (cross-build)
  - armv6l-musl (cross-build)
  - armv7l (cross-build)
  - armv7l-musl (cross-build)
  - aarch64 (cross-build)
  - aarch64-musl (cross-build)

### CODE REVIEW

**Comment by ufud-org** [comment]
second try, now with proper template revision number and without screwed up branch ...

File: srcpkgs/zssh/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+depends="openssh inetutils-telnet lrzsz"
+short_desc="Zmodem aware telnet and ssh cli client"
+maintainer="Niels Haedecke <webmaster@ufud.org>"
+license="GPL-2.0-only"
+homepage="https://zssh.sourceforge.net/"
+distfiles="http://prdownloads.sf.net/zssh/zssh-${version}.tgz"
+checksum="a2e840f82590690d27ea1ea1141af509ee34681fede897e58ae8d354701ce71b"
+
+pre_build() {
+	rm -f ${XBPS_WRAPPERDIR}/strip
```
[the remark this answers is not in the document]
Comment (ufud-org) [reply srcpkgs/zssh/template:16]: Makefile uses strip with additional parameter which is not supported by wrapper script, thus not creating stripped binatied. Found this solution to work around this issue while reading the voidlinux subreddit threads and other templates so assumed this to be proper way to do it.

[the remark this answers is not in the document]
Comment (classabbyamp) [reply srcpkgs/zssh/template:16]: the wrapper supports all arguments, the whole point of it is to ignore all strip invocations so xbps-src can strip things itself

File: srcpkgs/zssh/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+do_install() {
+	vmkdir usr/bin
+	vcopy zssh usr/bin/zssh
+	vcopy ztelnet usr/bin/ztelnet
```
[the remark this answers is not in the document]
Comment (ufud-org) [reply srcpkgs/zssh/template:22]: will change and add missing vman commands as well to provide proper man files

File: srcpkgs/zssh/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (ufud-org) [reply srcpkgs/zssh/template:16]: I see. I think this is the part I've misunderstood. Will remove as recommended.

**Comment by paper42** [comment]
Make sure this PR contains one commit per package as mentioned in CONTRIBUTING.md

**Comment by ufud-org** [comment]
> Make sure this PR contains one commit per package as mentioned in CONTRIBUTING.md

just to clarify - what do you mean by 'per package' ?

**Comment by paper42** [comment]
You are touching one package, so this PR should contain one commit with the appropriate commit message which you can find in CONTRIBUTING.md. Please fix this in this PR and don't open a new one. In this state it can not be merged because it doesn't meet our quality requirements, but that can be fixed with a force push.

**Comment by ufud-org** [comment]
> You are touching one package, so this PR should contain one commit with the appropriate commit message which you can find in CONTRIBUTING.md. Please fix this in this PR and don't open a new one. In this state it can not be merged because it doesn't meet our quality requirements, but that can be fixed with a force push.

Could you please you point me to some documentation on how to accomplish this without the risk of doing something wrong?

File: srcpkgs/zssh/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+version=1.5c
+revision=1
+build_style=gnu-configure
+makedepends="readline-devel"
+depends="openssh inetutils-telnet lrzsz"
+short_desc="Zmodem aware telnet and ssh cli client"
+maintainer="Niels Haedecke <webmaster@ufud.org>"
+license="GPL-2.0-only"
+homepage="https://zssh.sourceforge.net/"
+distfiles="http://prdownloads.sf.net/zssh/zssh-${version}.tgz"
```
Comment (classabbyamp) [inline srcpkgs/zssh/template:12]:
```suggestion
distfiles="${SOURCEFORGE_SITE}/zssh/zssh-${version}.tgz"
```

Comment (paper42) [reply srcpkgs/zssh/template]: You will have to Google git basics

Comment (ufud-org) [reply srcpkgs/zssh/template]: @paper42 done - consolidated commit history. Is this okay now?

Comment (ufud-org) [reply srcpkgs/zssh/template]: fixed commit message to conform to CONTRIBUTING.md

**Pull request closed without merging**